numpy/lib/index_tricks.py

```diff
@@ -269,8 +269,9 @@ class OGridClass(nd_grid):
     the stop value **is inclusive**.
 
     Returns
-    ----------
-    mesh-grid `ndarrays` with only one dimension :math:`\\neq 1`
+    -------
+    mesh-grid
```

Comment:
Class documentation should not have a `Return` section. Might document the constant instances `ogrid` and `mgrid`.